shashi/jan25.js

```diff
@@ -0,0 +1,48 @@
+
+const isPalindrome = str => {
+  let strNew;
+  if (typeof str === "number") {
+    strNew = str.toString();
+  }
+  if (str.length === 1 || str.length === "" || str === " ") {
+    return -1;
+  }
+  strNew = (str.split(" ")).join("").toLowerCase();
```

Comment:
nice, but why is this inefficient using split and join? This is not wrong but I wanted to make sure you know what each of these methods is actually doing.

Reply:
split converts the string to an array with space as delimiter, join joins the array with no spacing, but for equality comparison we need the string to be in same case right?